specs/fulu/das-core.md

```diff
@@ -80,9 +80,9 @@ The following values are (non-configurable) constants used throughout the specif
 ```python
 class DataColumnSidecar(Container):
     index: ColumnIndex  # Index of column in extended matrix
-    column: List[Cell, MAX_BLOB_COMMITMENTS_PER_BLOCK]
-    kzg_commitments: List[KZGCommitment, MAX_BLOB_COMMITMENTS_PER_BLOCK]
-    kzg_proofs: List[KZGProof, MAX_BLOB_COMMITMENTS_PER_BLOCK]
+    column: List[Cell, MAX_BLOBS_PER_BLOCK_FULU]
+    kzg_commitments: List[KZGCommitment, MAX_BLOBS_PER_BLOCK_FULU]
+    kzg_proofs: List[KZGProof, MAX_BLOBS_PER_BLOCK_FULU]
```

Comment:
Thank you but we actually do want this to be the maximum value. So we don't need to redefine `DataColumnSidecar` each time we raise the blob limit constant. We check elsewhere that an appropriate number of sidecars is provided. This is similar to `BeaconBlockBody`:

https://github.com/ethereum/consensus-specs/blob/6c581ca42c7e98fbdee7beed5643a41b874100d2/specs/deneb/beacon-chain.md?plain=1#L121